src/containers/Search.js

```diff
@@ -102,56 +157,92 @@ class search extends React.Component {
-                  <Input type="checkbox" /> Department name
-                </Label>
-              </FormGroup>
-            </Form>
-          </Col>
-        </Row>
-      </ Container>
-    )
+          renderPageNumbers = pageNumbers.map(number => {
+            return (
+              <PaginationItem key={number} className={(this.state.currentPage === number ? 'active ' : '')}>
+                <PaginationLink id={number} onClick={this.handleClick}>
+                  {number}
+                </PaginationLink>  
+              </PaginationItem>
+            );
+          });
+            
+        }else{
+          results =  __('No result found');
+        } 
+            
+        const styleClasses = (!data)
+          ? 'search-results-none' : 'list-unstyled search-results';
+
+        return (
+          <Container>
+            <div className="search-bar">
+              <ProfileSearch />
+            </div>
+            <Row>
+              <Col xs="2" sm="2">
+                <Form>
+                  <FormGroup>
+                    <Label for="sort">{__('Sort by')}</Label>
+                    <Input type="select" onChange={(e) => this.handleAlphabetClick(e)} name="sort" id="sort">
+                      <option>---</option>
+                      <option value="desc">{__('Alphabetical')}</option>
+                      <option value="asc">{__('Unalphabetical')}</option>                  
+                    </Input>
+                  </FormGroup>
+                </Form>
+              </Col> 
+            </Row>
+            <Row>
+              <Col xs="12" sm="10">
+                <ul>{results}</ul>
+              </Col>
+              <Col className='filter-section'>
+                <h5>Filter</h5>
+                <Form>
+                  <FormGroup>
+                    <Label check>
+                      <Input type="checkbox" /> Internal
+                    </Label>
+                  </FormGroup>
+                  <FormGroup>
+                    <Label check>
+                      <Input type="checkbox" /> External
+                    </Label>
+                  </FormGroup>
+                  <FormGroup>
+                    <Label check>
+                      <Input type="checkbox" /> Department name
+                    </Label>
+                  </FormGroup>
+                </Form>
+              </Col>
+            </Row>
+            <Row>
+              <Col xs="12" sm="10">
+                <Pagination style={{display: 'flex', justifyContent: 'center'}} aria-label="Page navigation" id="page-numbers">
+                  <PaginationItem>
+                    <PaginationLink onClick={() =>{this.next_save(currentPage-1,pageNumbers)}} > {"<"} </PaginationLink>
+                  </PaginationItem>
+                  {renderPageNumbers}
+                  <PaginationItem>
+                    <PaginationLink onClick={() =>{this.next_save(currentPage+1,pageNumbers)}} >{">"} </PaginationLink>
```

Comment:
Can we switch the < / > buttons to Previous / Next? That should give more context to what the button does.